checkbox-radio.scss

```diff
@@ -14,20 +14,20 @@
 	& > input[type="#{$type}"] {
 		margin-top: -0.2rem;
 		margin-right: 0.6rem;
-		margin-left: -1.7rem;
+		margin-left: -1.6rem;
```

Comment:
After looking again at some of the MVC tests, I think the original (-1.7) is most correct, to make sure these checkboxes line up vertically with checkboxes that aren't in containers (eg: 'select all' checkboxes). They might crowd things to the right a little bit (2px), but that seems less bad than vertical misalignment.